### PR TITLE
[116] Remove table headers in results when no submissions

### PR DESCRIPTION
**Pull request by czorek**

### CODE REVIEW

File: app/views/submissions/results.html.erb [hunk 1]
```diff
@@ -4,7 +4,7 @@
 <aside class="aside-widget aside-widget-fixed aside-widget-dashboard">
   <h2 class="aside-widget-header">Download:</h2>
   <ul class="download-list">
-    <li><%= link_to "Accepted", csv_download_accepted_path %></li>
-    <li><%= link_to "Waitlist", csv_download_waitlist_path %></li>
+    <li><%= link_to_if !submissions_accepted.empty?, "Accepted", csv_download_accepted_path %></li>
```
Comment (przemyslawhaptas) [inline app/views/submissions/results.html.erb]: What happens if it is empty? Does it leave the enclosing `<li></li>` element?

File: spec/features/results_download_spec.rb [hunk 2]
```diff
@@ -12,8 +17,6 @@
   it "visits results page, clicks 'Waitlist', checks if csv file is downloaded" do
```
Comment (przemyslawhaptas) [inline spec/features/results_download_spec.rb]: It doesn't, it only visits results page and checks if there is a link with that text. ;)

Comment (przemyslawhaptas) [inline spec/features/results_download_spec.rb]: As this scope describes "csv downloading process", I'd not check if there is this selector but try to click it like in the first spec.

File: spec/features/results_download_spec.rb [hunk 3]
```diff
@@ -12,8 +17,6 @@
...
     login_as(user, scope: :user)
     visit submissions_results_path
-    click_link('Waitlist')
-    expect(response_headers["Content-Type"]).to eq("text/csv")
-    expect(response_headers["Content-Disposition"]).to eq("attachment; filename=\"waitlist.csv\"")
+    expect(page).to have_selector('li', text: 'Waitlist')
   end
```
Comment (przemyslawhaptas) [inline spec/features/results_download_spec.rb]: I'd also add another describe with two specs for disappearing links when the lists are empty. ;)

File: app/views/submissions/results.html.erb (hunk 1 above)
Comment (czorek) [inline app/views/submissions/results.html.erb]: It's just an inactive link then - the element stays, but the text is black and unclickable

File: spec/features/results_download_spec.rb (hunk 2 above)
Comment (czorek) [inline spec/features/results_download_spec.rb]: Why would we try to click it? I don't think that's possible, if it's just a text basically

Comment (przemyslawhaptas) [inline spec/features/results_download_spec.rb]: I mean, click the link to `Waitlist` just as we click the link to `Accepted` above. ;)

File: spec/features/results_download_spec.rb (hunk 3 above)
Comment (czorek) [inline spec/features/results_download_spec.rb]: So having two tests for active links and two for inactive ones? Sounds like overkill for me ;p

Comment (przemyslawhaptas) [inline spec/features/results_download_spec.rb]: Well there is some logic that's not tested. Your call. ;)

**Pull request merged**